### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by CarlosHWolox**
## Summary
Added new tech cards and layout with flexbox 

## Screenshot
captura de pantalla de 2018-05-30 12-34-13

## Trello Card
https://trello.com/c/QAEBxy6X/23-content-layout

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.tech-border {
+    border: 1px solid $green;
+    border-radius: 5px;
+    height: 350px;
+    margin: 10px;
+    width: 200px;
+    
+    .tech-data {
```
Comment (HarryPerez) [inline scss/tech.scss]: Check identation in the whole file

Comment (HarryPerez) [reply scss/tech.scss]: Remember separating parent from children should be `2 spaces or 1 tab`

**Review by HarryPerez — CHANGES_REQUESTED** [review]

File: scss/styles.scss [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+
+body {
+    padding: 2% 10%;
+}
+
+.tech-container{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding: 0 2%;
```
Comment (HarryPerez) [inline scss/styles.scss]: use px. I think `padding: 0 120px` should be fine. What do you think?

**Review by HarryPerez — CHANGES_REQUESTED** [review]

File: scss/tech.scss [hunk 3]
```diff
@@ -1,5 +1,8 @@
 .tech-container {
   padding: 10px 40px;
```
Comment (HarryPerez) [inline scss/tech.scss]: check the order of the styles. Order them `alphabetically`

**Review by HarryPerez — CHANGES_REQUESTED** [review]

File: scss/tech.scss [hunk 4]
```diff
@@ -1,5 +1,8 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```
Comment (SKOLZ) [inline scss/tech.scss]: margin in each `tech` should be enough. you can reduce the 1280px width if we want less items per row

File: scss/tech.scss [hunk 5]
```diff
@@ -1,4 +1,7 @@
 .tech-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```
Comment (SKOLZ) [inline scss/tech.scss]: margin in each `tech` should be enough. you can reduce the 1280px width in the container if we want less items per row

**Review by SKOLZ — CHANGES_REQUESTED** [review]

**Review by HarryPerez — APPROVED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**